src/waveform/waveformwidgetfactory.cpp

```diff
@@ -94,27 +94,26 @@ WaveformWidgetHolder::WaveformWidgetHolder(WaveformWidgetAbstract* waveformWidge
 
 ///////////////////////////////////////////
 
-WaveformWidgetFactory::WaveformWidgetFactory() :
-        m_type(WaveformWidgetType::Count_WaveformwidgetType),
-        m_config(0),
-        m_skipRender(false),
-        m_frameRate(30),
-        m_endOfTrackWarningTime(30),
-        m_defaultZoom(WaveformWidgetRenderer::s_waveformDefaultZoom),
-        m_zoomSync(false),
-        m_overviewNormalized(false),
-        m_openGlAvailable(false),
-        m_openGlesAvailable(false),
-        m_openGLShaderAvailable(false),
-        m_beatGridAlpha(90),
-        m_vsyncThread(NULL),
-        m_pGuiTick(nullptr),
-        m_pVisualsManager(nullptr),
-        m_frameCnt(0),
-        m_actualFrameRate(0),
-        m_vSyncType(0),
-        m_playMarkerPosition(WaveformWidgetRenderer::s_defaultPlayMarkerPosition) {
-
+WaveformWidgetFactory::WaveformWidgetFactory()
+        : m_type(WaveformWidgetType::Count_WaveformwidgetType),
+          m_config(0),
```

Comment:
can we move these initializations to where the members are defined in the header for the class, like

protected:  // or whatever
 int m_frameRate = 30; 

?

Reply:
We have decided to not initialize the member variable in the header, because these values are out of sight. 
https://www.mixxx.org/wiki/doku.php/coding_guidelines#non-static_data_member_initializers

Reply:
sounds good